Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentSubgraph.php

```diff
@@ -305,6 +307,43 @@ public function findSubtree(NodeAggregateId $entryNodeAggregateId, FindSubtreeFi
         return $rootSubtrees->first();
     }
 
+    public function findAncestorNodes(NodeAggregateId $entryNodeAggregateId, FindAncestorNodesFilter $filter): Nodes
+    {
+        [
```

Comment:
As we are trying to enforce type safty general, a DTO would be a better fit IMHO. But maybe overengineering for a private/internal method.